resources/schema.gql

```diff
@@ -3065,28 +3107,205 @@ type ModifyPasswordWrongOldPassword implements ModifyPasswordResult {
 
 type Molecule {
 	"""
-	Looks up the project
+	Looks up the project.
+	
+	Shortcut for `v1 { project }`.
 	"""
-	project(ipnftUid: String!): MoleculeProject
+	project(ipnftUid: String!): MoleculeProject @deprecated(reason: "Use `v2 { project }` instead.")
 	"""
-	List the registered projects
+	List the registered projects.
+	
+	Shortcut for `v1 { projects }`.
 	"""
-	projects(page: Int, perPage: Int): MoleculeProjectConnection!
+	projects(page: Int, perPage: Int): MoleculeProjectConnection! @deprecated(reason: "Use `v2 { projects }` instead.")
 	"""
 	Latest activity events across all projects in reverse chronological
-	order
+	order.
+	
+	Shortcut for `v1 { activity }`.
 	"""
-	activity(page: Int, perPage: Int): MoleculeProjectEventConnection!
+	activity(page: Int, perPage: Int): MoleculeProjectEventConnection! @deprecated(reason: "Use `v2 { activity }` instead.")
+	"""
+	1-st Molecule API version (query).
+	"""
+	v1: MoleculeV1! @deprecated(reason: "Use `v2` instead")
+	"""
+	2-nd Molecule API version (query).
+	"""
+	v2: MoleculeV2!
+}
+
+input MoleculeActivityFiltersV2 {
+	"""
+	This filter is provided as an example.
+	"""
+	byIpnftUids: [String!]
+}
+
+type MoleculeAnnouncementEntryV2 {
+	project: MoleculeProjectV2!
+	id: String!
+	headline: String!
+	body: String!
+	attachments: [String!]!
+	accessLevel: String!
+	changeBy: AccountID!
+	categories: [String!]!
+	tags: [String!]!
+}
+
+type MoleculeAnnouncementsDatasetMutV2 {
+	"""
+	Creates an announcement record for the project.
+	"""
+	create(		headline: String!,		body: String!,
+		"""
+		List of dataset DIDs to link
+		"""
+		attachments: [String!],		moleculeAccessLevel: String!,		moleculeChangeBy: String!,		categories: [String!]!,		tags: [String!]!
+	): CreateAnnouncementResult!
+}
+
+type MoleculeAnnouncementsDatasetV2 {
+	"""
+	Access the underlying core Dataset
+	"""
+	dataset: Dataset!
+	byId(id: String!): MoleculeAnnouncementEntryV2!
```

Comment:
This type is missing `tail()` endpoint to allow listing the latest announcements with filters